ui/components/src/EChart/EChart.stories.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
What's the difference between Meta and StoryObj? (I was trying to find in the Storybook docs but wasn't successful, I guess I could plunge into their Github repo too)

Reply:
`Meta` is for the overall configuration for a given storybook file and `StoryObj` is for configuring a specific story. Lemme see if I can dig up the docs on this. I think they might be in a blog post more than in the general docs because they're still finishing up the full docs for the v7 beta...

Reply:
There's some bits in https://storybook.js.org/blog/improved-type-safety-in-storybook-7/ (altho we're not using all the `satisfies` bits they mention). There's also a bunch of good examples in https://storybook.js.org/docs/7.0/react/writing-stories/introduction if you click the "React (TS)" tabs in the examples.